raiden/transfer/node.py

```diff
@@ -845,6 +848,20 @@ def handle_receive_transfer_refund_cancel_route(
     return subdispatch_to_paymenttask(chain_state, state_change, new_secrethash, storage)
 
 
+def handle_receive_transfer_refund_cancel_route_light(
+    chain_state: ChainState, state_change: ActionTransferRerouteLight, storage
```

Comment:
ActionTransferRerouteLight is handled by this function at node.py lvl but by handle_transfer_reroute_light at initiator lvl, i dont understand that.

Reply:
i lost you here, can you explain more what you don't understand, baically ActionTransferRerouteLight was missing and it wasn't handled, you told me to add this to handle that to make the balance proof to be updated.

Reply:
We solve this in the google meet, i removed the clone code and the secret hash as we spoke